src/pages/Documents/columns.tsx

```diff
@@ -40,6 +40,7 @@ const columns = [
   {
     title: "Додатки",
     dataIndex: "fileName",
+    align: 'center' as const,
```

Comment:
Why did you write "as const"? I'm wondering)

Reply:
got the same problem as here https://stackoverflow.com/questions/61519622/antd-with-typescript-table-with-column-align-right-is-not-compiling